packages/strapi-plugin-content-manager/config/queries/mongoose.js

```diff
@@ -308,9 +309,10 @@ module.exports = {
 
     // Push new relation to the association array.
     value.push({
-      ref: params.refId,
+      ref: params.ref,
+      refId: params.refId,
       kind: params.ref,
-      field: association.filter
+      field: params.field
```

Comment:
The `field` attribute is customisable so I would suggest to create the object like that 

```js
value.push({
  ref: params.ref,
  refId: params.refId,
  kind: params.ref,
  [association.filter]: params.field
});
```

I know this as nothing to do with the specific PR but it could avoid issues in the future...